docs/integrations/google-cloud-storage/gcs.md

```diff
@@ -0,0 +1,57 @@
+---
+id: googlegcs
+title: Google Cloud Storage
+sidebar_label: Google Cloud Storage Data
+# prettier-ignore
+description: Setting up an integration with Google Cloud Storage
```

Comment:
Ah old up, this needs to be added to `microsite/sidebars.json` for it to show up afaik

Reply:
@Rugvip thanks! good catch. Fixed!